.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,12 @@
-*Issue #, if available:*
+**Issue number:**
 
-*Description of changes:*
+Closes #
 
+**Description of changes:**
 
 
-By submitting this pull request, I confirm that my contribution is made under the terms of the licenses outlined in the LICENSE-SUMMARY file.
\ No newline at end of file
+**Terms of contribution:**
+
+By submitting this pull request, I agree that this contribution is
+dual-licensed under the terms of both the Apache License, version 2.0, and the
```

Comment:
OK. This part is incorrect. :)

See the `LICENSE-SUMMARY` file. This repo has 3 different licenses and MIT isn't even one of them. 

From the current `LICENSE-SUMMARY` file:
```
Site templates, source code, and stylesheets are made available under the Apache 2.0 license. See the LICENSE file.

The content and documenation is made available under the Creative Commons Attribution-ShareAlike 4.0 International License. See the LICENSE-CONTENT file.

The sample code within this documentation is made available under a modified MIT-0 license. See the LICENSE-SAMPLECODE file.
```

Reply:
Skimmed over too fast. Fixed that now.